sample/device-code-sample.js

```diff
@@ -93,8 +97,20 @@ context.acquireUserCode(resource, sampleParameters.clientId, 'es-mx', function (
                 console.log(err);
             }
             else {
-                console.log(tokenResponse);
+                authorityUrl = sampleParameters.authorityHostUrl + '/' + sampleParameters.anothertenant;
+                
+                var context2 = new AuthenticationContext(authorityUrl, null, cache);
```

Comment:
this is for the cross tenant refresh token testing, i want to show the way refresh tokens works for cross tenant refresh token.